adflow/pyADflow.py

```diff
@@ -3651,7 +3651,7 @@ def computeJacobianVectorProductFwd(self, xDvDot=None, xSDot=None, xVDot=None, w
         Returns
         -------
         dwdot, funcsdot, fDot : array, dict, array
-            One or more of the these are return depending on the \*Deriv flags
+            One or more of the these are return depending on the \\*Deriv flags
```

Comment:
Why do we need to escape the `*` character? Does it work without any slashes?

Reply:
The `*` is an inline emphasis character which then starts but does not end. It will create an undesired behavior.

Reply:
Ah I see. In that case would it make more sense to surround it with backticks? i.e. ``*Deriv``

Reply:
Sure, that would work if we want this to be literal. Actually many other things in the docstring there should be done this way too. I will update this.